### PR TITLE
Update managing-holds.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note for Enabling This feature via PS

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2484

File: microsoft-365/compliance/managing-holds.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ To view the custodian hold policy:
 
 4. On the flyout page, you can see hold statistics for the policy. You can also perform actions like apply a query to your custodian-based hold. For more information about creating a hold query and using conditions, see [Keyword queries and search conditions for Content Search](keyword-queries-and-search-conditions.md).
 
+> [!NOTE]
+> Custodial Holds cannot be turned ON or OFF from the Security & Compliance center Admin center. Aministrator needs to use Powershell to make this change. For more information, See [Set-CaseHoldPolicy](https://docs.microsoft.com/powershell/module/exchange/set-caseholdpolicy?view=exchange-ps)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/managing-holds.md]:
```suggestion
> Custodial holds cannot be turned on or off from the Security & Compliance admin center. Administrators need to use PowerShell to make this change. For more information, see [Set-CaseHoldPolicy](https://docs.microsoft.com/powershell/module/exchange/set-caseholdpolicy?view=exchange-ps).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@markjjo Please merge this PR. Thanks

**Comment by markjjo** [comment]
Closing this PR. I have offline thread about this issue with support engineer who made this request and the engineering team. I will re-address this when it's determined if documentation needs to be updagted.

**Comment by msbemba** [comment]
@markjjo Thank you for the update.

**Pull request closed without merging**